src/foam/u2/wizard/StepWizardletStepsView.js

```diff
@@ -213,7 +213,7 @@ foam.CLASS({
 
                       // Render title
                       .start('p').addClass(self.myClass('title'))
-                        .translate(wizardlet.capability.id+'.name', wizardlet.capability.name)
+                        .translate(wizardlet.id+'.name', wizardlet.title)
```

Comment:
these changes ..... here and in ScrollWizard, imply that we will not only use the capability.name on occasion but now also associating a name to the wizardlets....
also you'll need to define in locales.jrl the wizardlet.id.name for translations...but how is that really possible when these are generated ?https://github.com/kgrgreer/foam3/blob/9423f60a2be3cf299bd23cca49188dc3ccd17a93/src/foam/u2/wizard/BaseWizardlet.js#L54

Reply:
This is a conflict. For `CapabilityWizardlet`s this will have the same behaviour, so I don't think it breaks anything existing. However, _every non-capability wizardlet class is presently broken because of this_. `StepWizardletStepsView` should absolutely not be accessing the property `wizardlet.capability` unless it's first checking the subclass.

Reply:
ya so missed CapabilityWizardlet override